README.md

```diff
@@ -61,9 +73,46 @@ Publicly available MBTiles charts can be found from:
 
 Plugin adds support for `/resources/charts` endpoints described in [Signal K specification](http://signalk.org/specification/1.0.0/doc/otherBranches.html#resourcescharts):
 
-- `GET /signalk/v1/api/resources/charts/` returns metadata for all available charts
-- `GET /signalk/v1/api/resources/charts/${identifier}/` returns metadata for selected chart
-- `GET /signalk/v1/api/resources/charts/${identifier}/${z}/${x}/${y}` returns a single tile for selected offline chart. As charts-plugin isn't proxy, online charts is not available via this request. You should look the metadata to find proper request.
+- Return metadata for all available charts
+
+```bash
+# v1 API
+GET /signalk/v1/api/resources/charts/` 
+
+# v2 API
+GET /signalk/v2/api/resources/charts/` 
+```
+
+- Return metadata for selected chart
+
+```bash
+# v1 API
+GET /signalk/v1/api/resources/charts/${identifier}` 
+
+# v2 API
+GET /signalk/v2/api/resources/charts/${identifier}` 
+```
+
+#### Chart Tiles
+Chart tiles are retrieved using the url defined in the chart metadata.
+
+For chart files placed in the path(s) defined in the plugin configuration, the url will be:
+
+```bash
+/chart-tiles/${identifier}/${z}/${x}/${y}
```

Comment:
One of the ideas for using `/signalk` prefix is that one could use a reverse proxy that proxies not only signalk but also other services. This does not hold 100% for the admin ui and other stuff, but I would rather keep this under `/signalk` than at the root.

Reply:
The intent was to not have a chart plugin serve chart tiles under the `/signalk/v2/api/resources/charts` path.
Have shifted the path to be under `/signalk`.